docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -2714,6 +2714,31 @@ When running locally, PacketFence provides a basic `suricata.yaml` that can be m
 
 In the case that Suricata is running on a separate server, Suricata configuration will have to be handled separately, which is not the purpose of the present guide.
 
+The following section parameters are required under suricata.yaml to have proper md5 checksuming and filetype recognition:
```

Comment:
See if we should merge some part of the existing documentation
(especially around https://packetfence.org/doc/PacketFence_Administration_Guide.html#_blocking_malicious_activities_with_violations Suricata section)
There a NOTE referring to some external doc that explain exactly that